<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>es_mainwindowClass</class>
 <widget class="QMainWindow" name="es_mainwindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_V">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QEidosTreeWidget" name="EidosTreeWidget">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Arial&quot;;</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::AllEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
       <column>
        <property name="text">
         <string>Наименование</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Parent_ID</string>
        </property>
       </column>
      </widget>
      <widget class="QSplitter" name="splitter_H">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTableWidget" name="tableWidget_EAs"/>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Свойства</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QRadioButton" name="radioButton_BFH">
             <property name="text">
              <string>Атрибут типа</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_BFP">
             <property name="text">
              <string>Атрибут экземпляра</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Тип атрибута</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_Type">
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Имя поля</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_FieldName"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Заголовок</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Caption"/>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_alternated">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Псевдоним</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_Temporality">
           <property name="text">
            <string>Временной ряд</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QCheckBox" name="checkBox_Required">
             <property name="text">
              <string>Обязательный</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_Visible">
             <property name="text">
              <string>Видимый</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_Locked">
             <property name="text">
              <string>Закрыт</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_DLL">
           <property name="title">
            <string>DLL</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_17">
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Имя библиотеки</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_DLL_Name"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_18">
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Процедура/функция</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_DLL_Proc"/>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_LNK">
           <property name="title">
            <string>Ссылка</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QCheckBox" name="checkBox_Multiple">
              <property name="text">
               <string>Множественная</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_20">
              <item>
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Род</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_Species"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_21">
              <item>
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Класс</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_Eidos">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton_ChooseEidos">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_Hipotesys">
              <item>
               <widget class="QLabel" name="label_LnkHyp">
                <property name="text">
                 <string>Тип</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_Hipotesys">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton_ChooseHipotesys">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_NeedList">
              <property name="text">
               <string>Выводить списком</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Класс</string>
    </property>
    <addaction name="action_Eidos_add"/>
    <addaction name="action_Eidos_del"/>
    <addaction name="action_Eidos_rename"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Список атрибутов</string>
    </property>
    <addaction name="action_AddFilter"/>
    <addaction name="action_RemoveFilter"/>
    <addaction name="action_EA_Add"/>
    <addaction name="action_EA_Del"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Текущий атрибут</string>
    </property>
    <addaction name="action_AltCaption"/>
    <addaction name="action_SaveChanges"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Динамические библиотеки</string>
    </property>
    <addaction name="action_DLL_manager"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_quit"/>
   <addaction name="action_Eidos_add"/>
   <addaction name="action_Eidos_del"/>
   <addaction name="action_Eidos_rename"/>
   <addaction name="separator"/>
   <addaction name="action_EA_Add"/>
   <addaction name="action_EA_Del"/>
   <addaction name="separator"/>
   <addaction name="action_AltCaption"/>
   <addaction name="action_SaveChanges"/>
  </widget>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="../images/resources.qrc">
     <normaloff>:/PICS/exit.png</normaloff>:/PICS/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выйти из формы</string>
   </property>
   <property name="toolTip">
    <string>Закрыть окно Eidos studio</string>
   </property>
  </action>
  <action name="action_Eidos_add">
   <property name="icon">
    <iconset resource="../images/resources.qrc">
     <normaloff>:/PICS/lavr_add.png</normaloff>:/PICS/lavr_add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить потомка</string>
   </property>
   <property name="toolTip">
    <string>Добавить потомка </string>
   </property>
  </action>
  <action name="action_Eidos_del">
   <property name="icon">
    <iconset resource="../images/resources.qrc">
     <normaloff>:/PICS/lavr_del.png</normaloff>:/PICS/lavr_del.png</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="toolTip">
    <string>Удалить выделенный элемент класса</string>
   </property>
  </action>
  <action name="action_Eidos_rename">
   <property name="icon">
    <iconset resource="../images/resources.qrc">
     <normaloff>:/PICS/Leaf_rename.png</normaloff>:/PICS/Leaf_rename.png</iconset>
   </property>
   <property name="text">
    <string>Переименовать</string>
   </property>
  </action>
  <action name="action_AddFilter">
   <property name="icon">
    <iconset resource="../images/resources.qrc">
     <normaloff>:/PICS/filter2.png</normaloff>:/PICS/filter2.png</iconset>
   </property>
   <property name="text">
    <string>Установить фильтр</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_RemoveFilter">
   <property name="icon">
    <iconset resource="../images/resources.qrc">
     <normaloff>:/PICS/filteroff.png</normaloff>:/PICS/filteroff.png</iconset>
   </property>
   <property name="text">
    <string>Сбросить фильтр</string>
   </property>
  </action>
  <action name="action_EA_Add">
   <property name="icon">
    <iconset resource="../images/resources.qrc">
     <normaloff>:/PICS/edit_add.png</normaloff>:/PICS/edit_add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить новый атрибут</string>
   </property>
  </action>
  <action name="action_EA_Del">
   <property name="icon">
    <iconset resource="../images/resources.qrc">
     <normaloff>:/PICS/cancel.png</normaloff>:/PICS/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Удалить текущий атрибут</string>
   </property>
  </action>
  <action name="action_AltCaption">
   <property name="icon">
    <iconset resource="../images/resources.qrc">
     <normaloff>:/PICS/box_juliane_krug_s.png</normaloff>:/PICS/box_juliane_krug_s.png</iconset>
   </property>
   <property name="text">
    <string>Установить псевдоним заголовка</string>
   </property>
  </action>
  <action name="action_SaveChanges">
   <property name="icon">
    <iconset resource="../images/resources.qrc">
     <normaloff>:/PICS/apply.png</normaloff>:/PICS/apply.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить изменения</string>
   </property>
  </action>
  <action name="action_DLL_manager">
   <property name="text">
    <string>Менеджер библиотек</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEidosTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>qeidostreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/resources.qrc"/>
 </resources>
 <connections/>
</ui>
